### DISCUSSION TITLE
VRP Collect Solutions

### DISCUSSION THREAD

**Discussion by rahulj2409** [google/or-tools#2687] [Routing (and legacy CP) questions]
Is there a way to collect all of the solutions found by a VRP in google Or-tools?

I'm using log_search = True, but that only returns the one solution. (It's finding it in 5 minutes, but when I run it for 4 hours, it still just returns one solution).

Is there anything else I should be trying?

**Comment by lperron** [comment]
if it only prints one solution, maybe it has only found one solution.
Can you try on an smaller/easier problem?

Otherwise, you need to use a solution collector for that. There are examples lying around.
